pyproject.toml

```
@@ -185,18 +185,13 @@ minversion = "6.0"
 addopts = "-ra  --strict-config  --strict-markers"
```

```suggestion
addopts = ["-ra", "--strict-config", "--strict-markers", "--maxfail=5"]
```

If multiple tests fail, logs often do not provide additional information, only more scrolling.